jerry-core/ecma/builtin-objects/ecma-builtin-object.c

```diff
@@ -94,6 +97,67 @@ ecma_builtin_object_dispatch_construct (const ecma_value_t *arguments_list_p, /*
   }
 } /* ecma_builtin_object_dispatch_construct */
 
+#ifndef CONFIG_DISABLE_ES2015_BUILTIN
+/**
+ * [[GetPrototypeOf]]
+ *
+ * See also:
+ *          ES2015 9.1.1
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value
+ */
+static ecma_value_t
+ecma_builtin_object_object_get_prototype_of (ecma_value_t this_arg,
```

Comment:
Can't we just merge the new code path into the other `ecma_builtin_object_object_get_prototype_of` method? Do we really need to copy the whole function?